OsProbeTests#testOsStats fails when you don't have much swap free

Top says:

```
MiB Mem :  31715.4 total,   1456.4 free,  26600.9 used,   3658.1 buff/cache
MiB Swap:   4096.0 total,     21.1 free,   4074.9 used.   3770.5 avail Mem 
```

The test says:
```
java.lang.AssertionError: 
Expected: a value greater than <0L>
     but: <0L> was equal to <0L>
	at __randomizedtesting.SeedInfo.seed([880A718A8C675F27:300738D0F8DF317]:0)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at org.elasticsearch.monitor.os.OsProbeTests.testOsStats(OsProbeTests.java:126)
```

So, yeah, I should totally turn off some stuff before the OOKiller starts knocking out processes, but maybe the test shouldn't fail?

--- a/server/src/test/java/org/elasticsearch/monitor/os/OsProbeTests.java
+++ b/server/src/test/java/org/elasticsearch/monitor/os/OsProbeTests.java
@@ -123,7 +123,7 @@ public class OsProbeTests extends ESTestCase {
         long total = stats.getSwap().getTotal().getBytes();
         if (total > 0) {
             assertThat(stats.getSwap().getTotal().getBytes(), greaterThan(0L));
-            assertThat(stats.getSwap().getFree().getBytes(), greaterThan(0L));
+            assertThat(stats.getSwap().getFree().getBytes(), greaterThanOrEqualTo(0L));
             assertThat(stats.getSwap().getUsed().getBytes(), greaterThanOrEqualTo(0L));
         } else {
             // On platforms with no swap

